modules/oe_theme_content_organisation/oe_theme_content_organisation.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Implements hook_install().
+ */
+function oe_theme_content_organisation_install() {
+  // If we are installing from config, we bail out.
+  if (\Drupal::isConfigSyncing() === TRUE) {
+    return;
+  }
+
+  // Override entity view displays.
+  $config_name = 'core.entity_view_display.node.oe_organisation.default';
+  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_organisation') . '/config/overrides');
+  $config_values = $storage->read($config_name);
+
+  // Load existing configuration.
+  $config_manager = \Drupal::service('config.manager');
+  $entity_type = $config_manager->getEntityTypeIdByName($config_name);
+  $config_entity_storage = $config_manager
+    ->getEntityTypeManager()
+    ->getStorage($entity_type);
+  $config_entity = $config_entity_storage->load($config_values['id']);
+
+  // Update configuration.
+  $updated_config_entity = $config_entity_storage->updateFromStorageRecord($config_entity, $config_values);
+  $updated_config_entity->save();
```

Comment:
As we will need to provide a teaser for this content type too, let's use the same method we are using on all other content types, like on the event content type case below:

```php
function oe_theme_content_event_install() {
  // If we are installing from config, we bail out.
  if (\Drupal::isConfigSyncing() === TRUE) {
    return;
  }

  // Override entity view displays.
  $storage = new FileStorage(drupal_get_path('module', 'oe_theme_content_event') . '/config/overrides');
  $displays = [
    'core.entity_view_display.node.oe_event.default',
    'core.entity_view_display.node.oe_event.teaser',
  ];

  foreach ($displays as $display) {
    $values = $storage->read($display);
    $config = EntityViewDisplay::load($values['id']);
    if (!$config) {
      $config = EntityViewDisplay::create($values);
      $config->save();
      continue;
    }

    foreach ($values as $key => $value) {
      $config->set($key, $value);
    }

    $config->save();
  }
}
```

Reply:
done